src/Form/tests/FormAutosuggest.test.jsx

```diff
@@ -101,6 +101,25 @@ describe('FormAutosuggest', () => {
       expect(onSelected).toHaveBeenCalledTimes(1);
     });
 
+    it('console.log is called when onClick is passed', () => {
+      const logSpy = jest.spyOn(console, 'log');
```

Comment:
It would be better to create a separate mock function for `onClick` handler instead of hardcoding `console.log`, see for example how `onSelected` prop is tested in this file